app/views/layouts/_simple_autocomplete.html.haml

```diff
@@ -11,14 +11,15 @@
   tooltip = local_assigns[:tooltip].presence ||
             "Search for the intended #{association.to_s.humanize}, "\
             'then select it from the dropdown that appears below.'
-
+  input_name = local_assigns[:allow_new] ? "#{base_object}[#{association}_name]" : "#{association}_name_search"
```

Comment:
@calebtocco I wonder if we really need to rename it here. I kind of like it still being named search, its a little clearer what its doing and used for.

Reply:
true, we can leave the search and change the accessor